operator/tests/test-update.sh

```diff
@@ -9,11 +9,11 @@ readonly NAMESPACE="tailing-sidecar-system"
 readonly TIME=300
 
 wait_for_all_pods_running ${NAMESPACE} ${TIME}
+echo "$(kubectl describe pods -n ${NAMESPACE})"
```

Comment:
Is the `echo` needed here? Why not just do `kubectl describe pods -n ${NAMESPACE}`?

Reply:
It is not needed. Thanks for catching this!